bof-request.md

```diff
@@ -0,0 +1,43 @@
+# Name: Media over QUIC
+## Description
+ 
+Media over QUIC (moq) will work on solution for a simple low-latency 
+media delivery protocol that is common across ingest and distribution. 
+This work may address use cases including live streaming, gaming, and 
+media conferencing and allows efficient scaling. The solution supports 
+both web browsers and simple devices that don’t contain embedded browsers. 
+
+
+This is merely a placeholder for the BOF request, the details of the charter are  
+currently being worked developed, so please consider this as a heads up for 
+Media Over QUIC BOF request.
+
+Fill in the details below. Keep items in the order they appear here.
+
+## Required Details
+- Status: (not) WG Forming
+- Responsible AD: TBD
+- BOF proponents: TBD
+- BOF chairs: TBD
+- Number of people expected to attend: 70
+- Length of session (1 or 2 hours): 2 hours
+- Conflicts (whole Areas and/or WGs)
+   - Chair Conflicts: TBD
+   - Technology Overlap: TBD
+   - Key Participant Conflict: TBD
+
+## Information for IAB/IESG
+To allow evaluation of your proposal, please include the following items:
+
+- Any protocols or practices that already exist in this space:
+- Which (if any) modifications to existing protocols or practices are required:
+- Which (if any) entirely new protocols or practices are required:
+- Open source projects (if any) implementing this work:
+
+## Agenda
+   - TBD
+
+## Links to the mailing list, draft charter if any, relevant Internet-Drafts, etc.
+   - Mailing List: https://www.ietf.org/mailman/listinfo/moq
+   - Draft charter: https://github.com/moq-wg/moq-charter
+   - Relevant drafts: TBD
```

Comment:
I think I would leave this as TBD but at same time, list pretty much any draft in any way associated with this